playing-card-detector/legacyProject/CameraTester.py

```diff
@@ -0,0 +1,10 @@
+import cv2
```

Comment:
Is the playing-card-detector repo still used? Or will it be replaced with OpenCV? Anyway, we should consider just having it as a submodule if we need it. Otherwise, maybe we should remove this code from our repo if we're not using it?

Reply:
legacyProject is not longer in use, but will be useful as reference material for the new card detection